src/main/kotlin/cz/startnet/utils/pgdiff/CLI.kt

```diff
@@ -24,21 +25,33 @@ class CLI : CliktCommand(name = "apgdiff") {
     val oldDumpFile by argument(help = "Path to the original dump file").file(mustBeReadable = true)
     val newDumpFile by argument(help = "Path to the new dump file").file(mustBeReadable = true)
 
+    val schemas: List<String> by option(
+        "-n",
+        "--schema",
+        help = "diff the specified schema(s) only",
```

Comment:
i'd add the same hint as pg_dump help does
https://www.postgresql.org/docs/12/app-pgdump.html#PG-DUMP-OPTIONS


```suggestion
        help = "diff the specified schema(s) only. diffs all if not given",
```